src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -55,14 +61,16 @@ public function load($data, $type = null)
 
         foreach ($this->resourceNameCollectionFactory->create() as $resourceClass) {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $normalizedShortName = Inflector::pluralize(Inflector::tableize($resourceMetadata->getShortName()));
+
+            $shortNamePath = $resourceMetadata->getShortName() ?? '';
```

Comment:
Right :stuck_out_tongue_winking_eye: nvm thought it was defaulted to a string but in fact it's null.